scipy/optimize/_linprog.py

```diff
@@ -339,13 +348,31 @@ def linprog(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None,
     This section describes the available solvers that can be selected by the
     'method' parameter.
 
+    :ref:`'highs-simplex' <optimize.linprog-highs-simplex>` and
```

Comment:
Since the links are working I am hesitant, but I will try for this paragraph initially.
With manual references:
![image](https://user-images.githubusercontent.com/6570539/87235968-b5745700-c397-11ea-8db6-4ff179849975.png)
_Update with single backticks when CircleCI finishes..._